.github/workflows/pr-clean-caches.yml

```diff
@@ -0,0 +1,40 @@
+name: Cleanup branch caches
+
+on:
+  pull_request:
+    types: [opened, closed, reopened, synchronize]
+  push:

```

Comment:
This is probaly not inteded. Your PR would delete cache
- on push,
- edit of description,
- sync
- …

Reply:
What else its doing? can't you see what types are given in it?